src/include/processor/result/result_set.h

```diff
@@ -4,7 +4,7 @@
 
 #include "common/data_chunk/data_chunk.h"
 #include "processor/data_pos.h"
-#include "storage/storage_structure/lists/list_sync_state.h"
+#include "storage/storage_structure/lists/list_handle.h"
```

Comment:
I think we no longer init list_handler in the result_set, we can remove this include

Reply:
Done.